### PR TITLE
Allow demos to be opened fullscreen in new tab

### LINKED ISSUE #294: cmd+click on a demo should open up in a new tab without the modal

### PR DESCRIPTION
**Pull request by samuelli**
Fixes #294

### CODE REVIEW

File: client/src/catalog-element.html [hunk 1]
```diff
@@ -707,7 +707,15 @@ <h2 title>[[subElement]]</h2>
         }
       },
 
-      _demoClicked: function() {
+      _demoClicked: function(event) {
+        var isMac = navigator.userAgent.indexOf('Mac OS X') != -1;
+        var mouseEvent = event.detail.sourceEvent;
+        if ((!isMac && mouseEvent.ctrlKey) || (isMac && mouseEvent.metaKey)) {
```
Comment (dstockwell) [inline client/src/catalog-element.html]: I think if ctrl/meta/anything is pressed we should just fall back to the default behavior. Seems less fragile than relying on the useragent.

**Comment by dstockwell** [comment]
> dstockwell approved these changes just now

oh, so that's what that button does.

File: client/src/catalog-element.html (hunk 1 above)
Comment (samuelli) [inline client/src/catalog-element.html]: Shouldn't we better match browser behavior?

**Comment by samuelli** [comment]
Pending comments on #294

**Pull request closed without merging**